### PR TITLE
BorrowBookControl ready for review

### PR DESCRIPTION
**Pull request by jazzie-b**

### CODE REVIEW

**Comment by TwistedHead** [comment]
All code style guidelines have been followed. I am OK for it to be merged with the Master.

File: src/BorrowBookControl.java [hunk 1]
```diff
@@ -3,114 +3,112 @@
...
+			state = ControlState.SCANNING; 
+		} else {
 			ui.display("Member cannot borrow at this time");
-			ui.setState(BorrowBookUI.UI_STATE.RESTRICTED); }}
+			ui.setState(BorrowBookUi.UiState.RESTRICTED); 
+		}
+	}
 	
 	
 	public void Scanned(int bookId) {
```
Comment (pyroib) [inline src/BorrowBookControl.java]: check case for this method name

5.2.3 Method names
Method names are written in lowerCamelCase.

Comment (jazzie-b) [reply src/BorrowBookControl.java]: Method name has been changed

File: src/BorrowBookControl.java [hunk 2]
```diff
@@ -2,114 +2,114 @@
...
-	
-	public void cancel() {
-		ui.setState(BorrowBookUi.UI_STATE.CANCELLED);
-		state = ControlState.CANCELLED;
-	}
-	
-	
+    
+    public void cancel() {
+        ui.setState(BorrowBookUi.UI_STATE.CANCELLED);
```
Comment (pyroib) [inline src/BorrowBookControl.java]: check UI_STATE -> UiState

**Comment by jazzie-b** [comment]
Fixes made to UiState

**Pull request merged**